includes/experiments/class-wc-calypso-bridge-task-list-reminderbar-experiment.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * AB Experiment handling for the reminder bar task list nudge.
+ *
+ * @package WC_Calypso_Bridge/Classes
+ * @since   x.x.x
+ * @version x.x.x
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Calypso_Bridge_Task_List_ReminderBar_Experiment Class.
+ */
+class WC_Calypso_Bridge_Task_List_ReminderBar_Experiment {
+
+	/**
+	 * The single instance of the class.
+	 *
+	 * @var object
+	 */
+	protected static $instance = null;
+
+	/**
+	 * Get class instance.
+	 *
+	 * @return object Instance.
+	 */
+	final public static function get_instance() {
+		if ( null === static::$instance ) {
+			static::$instance = new static();
+		}
+		return static::$instance;
+	}
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+
+		// Only in Ecommerce and Woo Express sites.
+		if ( ! wc_calypso_bridge_has_ecommerce_features() ) {
+			return;
+		}
+
+		if ( defined('REST_REQUEST') && REST_REQUEST ) {
+			return;
+		}
+
+		if ( ( defined( 'DOING_CRON' ) && DOING_CRON ) || ( defined('DOING_AJAX') && DOING_AJAX ) || ( defined( 'WP_CLI' ) && WP_CLI ) ) {
+			return;
+		}
+ 
+		$this->init();
+	}
+
+	/**
+	 * Init experiment.
+	 */
+	public function init() {
+
+		add_filter( 'pre_option_woocommerce_task_list_reminder_bar_hidden', function( $pre_option ) {
+			return self::is_experiment_treatment() ? $pre_option : 'yes';
+		} );
+	}
+
+    /**
+	 * Check if current session is experiment treatment.
+	 *
+	 * @return bool Returns true if the current session is treatment.
+	 */
+	protected static function is_experiment_treatment() {
```

Comment:
It looks like this is [forking](https://github.com/Automattic/woocommerce/blob/5661d563f502540781b6e609c685bfc8def4685a/plugins/woocommerce/includes/react-admin/class-experimental-abtest.php#L106) the method `in_treatment` in `Experimental_Abtest`. How come we don't call this method directly?

Reply:
Thanks for taking a look, @aaronyan! I actually took this part from other references in the WC codebase. I don't see why we shouldn't use this handy helper! Updated at https://github.com/Automattic/wc-calypso-bridge/pull/1219/commits/ff7eb0b8108cd5e0d80483baf8cd64cc6d8bbe8b